### ISSUE TITLE
[BUG] Prometheus exporter is wrong when array is used in configuration for FullKeepCnt

### ISSUE THREAD

**Issue by phoenix741** [backuppc/backuppc#462]
Hello,

I configure prometheus to scrap this URL: `http://192.168.101.205/backuppc/index.cgi?action=metrics&format=prometheus`

If i set FullKeepCnt to 
```perl
$Conf{FullKeepCnt} = [
  6,
  3,
  3,
  3
];
```

the prometheus metrics will be :

```
backuppc_hosts_full_keep_count{host="localhost"} ARRAY(0x55d9ea7df800)
```

This is an incorrect value, and the all the metrics can't be used.

I have another error too, if no metrics is available i have no value (instead of -1 or 0)

```
backuppc_hosts_incr_duration{host="pc-m-ulrich"}  
```

Thanks

**Comment by phoenix741** [comment]
I think this is fixed in "master" but not in 4.4.0:

https://github.com/backuppc/backuppc/blob/master/lib/BackupPC/CGI/Metrics.pm

I have patched my Metrics file and it's working.

**Comment by freelock** [comment]
Hi,

I was hitting this, and digging into it I see the patch here -- would be great to get this into a release...

But I think there's a better fix. The current fix just uses the first number in the array. In my fix, I used "sum" from List::Util to sum the numbers in the array, getting a count of the configured backups to keep.

**Comment by GWHAYWOOD** [comment]
Agreed that it would be great to get to the point where I can make a new release, and that's my focus at present.

In the meantime I think this is handled better now in master, but please take a look at the latest code to see if it meets your requirements.  If you have any comments please reply to [my post on the mailing list](https://sourceforge.net/p/backuppc/mailman/message/59249466/).  If necessary we can re-open this issue.
